biogeophys/FatesPlantHydraulicsMod.F90

```diff
@@ -3151,14 +3883,20 @@ subroutine bisect_pv(ft, pm, lower, upper, xtol, ytol, psi_node, th_node)
     real(r8) :: y_new                  ! corresponding y value at x.new
     real(r8) :: f_new                  ! y difference between new y guess at x.new and target y
     real(r8) :: chg                    ! difference between x upper and lower bounds (approach 0 in bisection)
+    integer  :: nitr                   ! number of iterations 
     !----------------------------------------------------------------------
-  
+    if(psi_node > 0.0_r8) then
+      write(fates_log(),*)'Error: psi_note become positive,&
+                           psi_node=',psi_node
```

Comment:
Please note that IBM compiler issues a build error on a string split into multiple lines without a continuation character on the next line: i.e. need to to update this to either
```
      write(fates_log(),*)'Error: psi_note become positive,&
                           & psi_node=',psi_node
```
or just 
```
      write(fates_log(),*)'Error: psi_note become positive, psi_node=',psi_node
```

Reply:
@amametjanov  this is something we can do and will try to enforce, thanks for bringing this up